environment-dev.yml

```diff
@@ -5,6 +5,11 @@ dependencies:
   - pandas==1.5.2
   - numpy>=1.18.5
   - pyarrow>=4.0.1
+  - ray-default>=1.4.0,<2.1.0
+  # This will allow us to install ray==2.0.1 instead of ray==1.12.1;
+  # Apparently the priority is to install a later version of protobuf, instead of ray
+  - protobuf>=3.15.3,<4.0.0
```

Comment:
This is a temporary pin, I suppose after we support the new version of ray (2.1.0), this can be removed.